package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "jade-knockout-template",
```

Comment:
Good idea. We are in node-land and I was trying to avoid a package.json by just doing stuff globally. This is the way to go as keeping deps local is nice. Being able to run the server without dipping into python-land is extra nice.